apiserver/facades/client/action/interface.go

```diff
@@ -62,6 +62,7 @@ func (s *stateShim) FindEntity(tag names.Tag) (state.Entity, error) {
 func (s *stateShim) Model() (Model, error) {
```

Comment:
On the `model` interface on this file can you please remove the following methods `Type` & `ModelTag`.